gym-service/src/main/java/org/tolking/controller/TraineeController.java

```diff
@@ -130,8 +139,8 @@ public List<TrainingTraineeReadDTO> getTrainingList(@NotNull Principal principal
         );
     }
 
-    @PostMapping("/training")
-    @ResponseStatus(HttpStatus.OK)
+    @org.springframework.web.bind.annotation.PostMapping("/training")
```

Comment:
Why full class names?

Reply:
Oh Sorry Recently, I have been testing import settings (to remove star import). And in process, some changes affected to code.